### DISCUSSION TITLE
Welcome to eulers-shield Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/eulers-shield#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YCJ999888** [comment]
314.159＄的价格是主网后的稳定价格 ，还是参考价格，谢谢！

**Reply by greicon** [reply]
Stable coin

**Reply by duongngogithub** [reply]
im not an it im not a coder its so hard to me to read all this infomation but i want to know more about pi. Thanks frens

**Comment by greicon** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
